### DISCUSSION TITLE
Translation way ?

### DISCUSSION THREAD

**Discussion by Tetsuosp** [freeplane/docs#97] [Q&A]
Hi guys,

Is this place the right place to talk about translation ?
I'm a long date french user of freeplane and want to promote it around me. But indeed, the best way - i think - is to install an entire software translate in natural user's language.
Actually, i work for my wife and translate first the "Basic Freeplane Functions" map, and want to work on "Tutorial Freeplane" map too.
Thx for your comments,
Regards

**Comment by euu2021** [comment]
In the documentation,[ there is a page](https://docs.freeplane.org/contribute/Translation_How-To.html) explaining how the workflow of the translation works. But, it seems that there is no translation for those tutorial maps that come bundled with FreePlane. I'm an enthusiast of the "freeplaneFunctions" map, so I think having translated versions of it would be nice.

**Comment by Tetsuosp** [comment]
Hi,

Right, i think so. That's why i'll try to do it :-)

To do so, i use [this page](https://docs.freeplane.org/contribute/Translation_How-To.html#translation-of-tutorial-or-documentation-maps). I hope to send my work in a couple of week ;p

Sent from [ProtonMail](https://protonmail.ch), encrypted email based in Switzerland.

------- Original Message -------
